dbt/adapters/athena/connections.py

```diff
@@ -97,6 +97,8 @@ def execute(
         endpoint_url: Optional[str] = None,
         cache_size: int = 0,
         cache_expiration_time: int = 0,
+        result_reuse_enable: Optional[bool] = None,
+        result_reuse_minutes: Optional[int] = None,
```

Comment:
I know that the impact is small but could you please move that into another PR? let's keep changes minimal in order to isolate possible future issues, as this is not related to LF

Reply:
resolved